frontend/packages/console-app/src/components/nodes/node-dashboard/queries.ts

```diff
@@ -0,0 +1,133 @@
+import * as _ from 'lodash';
+import { QueryWithDescription } from '@console/shared/src/components/dashboard/utilization-card/UtilizationItem';
+
+export enum NodeQueries {
+  CPU_USAGE = 'CPU_USAGE',
+  CPU_TOTAL = 'CPU_TOTAL',
+  MEMORY_USAGE = 'MEMORY_USAGE',
+  MEMORY_TOTAL = 'MEMORY_TOTAL',
+  POD_COUNT = 'POD_COUNT',
+  PODS_BY_CPU = 'PODS_BY_CPU',
+  PODS_BY_MEMORY = 'PODS_BY_MEMORY',
+  PODS_BY_FILESYSTEM = 'PODS_BY_FILESYSTEM',
+  PODS_BY_NETWORK_IN = 'PODS_BY_NETWORK_IN',
+  PODS_BY_NETWORK_OUT = 'PODS_BY_NETWORK_OUT',
+  FILESYSTEM_USAGE = 'FILESYSTEM_USAGE',
+  FILESYSTEM_TOTAL = 'FILESYSTEM_TOTAL',
+  NETWORK_IN_UTILIZATION = 'NETWORK_IN_UTILIZATION',
+  NETWORK_OUT_UTILIZATION = 'NETWORK_OUT_UTILIZATION',
+  NETWORK_UTILIZATION = 'NETWORK_UTILIZATION',
+  POD_RESOURCE_LIMIT_CPU = 'POD_RESOURCE_LIMIT_CPU',
+  POD_RESOURCE_LIMIT_MEMORY = 'POD_RESOURCE_LIMIT_MEMORY',
+  POD_RESOURCE_REQUEST_CPU = 'POD_RESOURCE_REQUEST_CPU',
+  POD_RESOURCE_REQUEST_MEMORY = 'POD_RESOURCE_REQUEST_MEMORY',
+}
+
+const queries = {
+  [NodeQueries.CPU_USAGE]: _.template(`instance:node_cpu:rate:sum{instance='<%= node %>'}`),
+  [NodeQueries.CPU_TOTAL]: _.template(`instance:node_num_cpu:sum{instance='<%= node %>'}`),
+  [NodeQueries.MEMORY_USAGE]: _.template(
+    `node_memory_MemTotal_bytes{instance='<%= node %>'} - node_memory_MemAvailable_bytes{instance='<%= node %>'}`,
+  ),
+  [NodeQueries.MEMORY_TOTAL]: _.template(`node_memory_MemTotal_bytes{instance='<%= node %>'}`),
+  [NodeQueries.POD_COUNT]: _.template(`kubelet_running_pod_count{instance=~'<%= ipAddress %>:.*'}`),
+  [NodeQueries.FILESYSTEM_USAGE]: _.template(
+    `instance:node_filesystem_usage:sum{instance='<%= node %>'}`,
+  ),
+  [NodeQueries.FILESYSTEM_TOTAL]: _.template(
+    `node_filesystem_size_bytes{mountpoint="/", instance='<%= node %>'}`,
```

Comment:
This is total filesystem size only for `/` mountpoint and not for all node filesystems. Is this intended?

Reply:
no, Im not sure why I added it there.. will be removed